edx_proctoring/templates/proctoring/seq_proctored_exam_footer.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+<div class="faq-proctoring-exam">
+  <h4> {% trans "See Also" %} </h4>
+    <p>
+    {% blocktrans %}
+      <a class="footer-link" href="#">
+        Frequently asked questions about proctoring and earning college credit.
+      </a>
+      <a class="footer-link" href="#">
+        Technical Requirements for taking a proctored exam
```

Comment:
Can we add a 3rd "See Also" item:
"edX's Online Proctoring Rules" link text with the target: http://edx.readthedocs.org/projects/edx-guide-for-students/en/latest/sfd_proctoredexams/SFD_OnlineProctoringPolicy.html